src/yunohost/backup.py

```diff
@@ -49,111 +51,199 @@
 
 BACKUP_PATH = '/home/yunohost.backup'
 ARCHIVES_PATH = '%s/archives' % BACKUP_PATH
+MARGIN_SPACE_SIZE = 100
 
 logger = getActionLogger('yunohost.backup')
 
 
-def backup_create(name=None, description=None, output_directory=None,
-                  no_compress=False, ignore_hooks=False, hooks=[],
-                  ignore_apps=False, apps=[]):
+class Archive:
```

Comment:
There's a class "Archive" and later a class "BackupArchive". The terminology is confusing ... isn't "Archive" also meant to handle .. backup archives too ?